src/parser.rs

```diff
@@ -1491,228 +1491,356 @@ impl<'input, Endian> Attribute<'input, Endian>
     ///
     /// See "Figure 20. Attribute encodings" and "Figure 21. Attribute form encodings".
     pub fn value(&self) -> AttributeValue<'input, Endian> {
-        // The following are already normalized: address, exprloc, flag, reference, string.
-        match self.name {
-            // DW_AT_sibling: reference
-            // DW_AT_location: exprloc, loclistptr
-            constants::DW_AT_location => {
+        macro_rules! address {
+            () => ();
+        }
+        macro_rules! block {
+            () => ();
+        }
+        macro_rules! constant {
```

Comment:
All these macros are nice, but it did take me a little bit to figure out what was going on, and how all the `() => ()` are nops/placeholders for future prettifying. I think a comment at the top of the method giving a short overview of the approach would be nice for readers.